extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/media/GoogleMediaImporter.java

```diff
@@ -143,6 +203,9 @@ public GoogleMediaImporter(
     this.connectionProvider = connectionProvider;
     this.monitor = monitor;
     this.writesPerSecond = writesPerSecond;
+    this.appCredentials = appCredentials;
+    this.retryingIdempotentExecutor = retryingIdempotentExecutor;
+    this.enableRetrying = enableRetrying;
```

Comment:
I dont see this field being used anywhere... I think in order to actually use the retrying executor, you need to use that instead of using the exeucotr thats passed into the importItem calls